factory/.env

```diff
@@ -11,16 +11,16 @@ BASE_IMAGE='debian:bullseye-slim'
 NODE_VERSION='18.14.1'
 
 # Update this to deploy the docker factory if you make changes to factory.Dockerfile or install scripts
-FACTORY_VERSION='2.0.0'
+FACTORY_VERSION='2.0.1'
 
 # Chrome versions: https://www.ubuntuupdates.org/package/google_chrome/stable/main/base/google-chrome-stable
-CHROME_VERSION='111.0.5563.64-1'
+CHROME_VERSION='111.0.5563.110-1'
```

Comment:
For reasons i can only guess at, it seems like when you install the minor minor (whatever you call the forth place on a version string) version of chrome and edge you only get the later version. So i've bumped the later version here to reflect that but it's something to consider for the factory in the future. (do we need to not allow minor minor?)